docs/ci.md

```diff
@@ -55,21 +57,21 @@ the E2E job configuration files found
 | `P` | Use the phased training within the 'full' training library |
 | `v` | Run with vLLM for serving |
 
-You can specify the following flag to test the large t-shirt size with the `e2e.sh`
-script. Support for the recommended workflow for other t-shirt sizes will be added to
-this script in the future.
+You can specify the following flags to test the medium and large t-shirt sizes with the `e2e-ci.sh`
+script. Support for the recommended workflow for the small t-shirt size will be added to this script in the future.
 
 | Flag | Feature |
 | --- | --- |
+| `m` | Run the e2e workflow for the medium t-shirt size of hardware |
 | `l` | Run the e2e workflow for the large t-shirt size of hardware |
 
 ### Current E2E Jobs
 
 | Name | T-Shirt Size | Runner Host | Instance Type | OS | GPU Type | Script | Flags | Runs when? | Slack reporting? |
-| --- | --- | --- | --- | --- | --- | --- | --- | --- |
-| [`e2e-nvidia-t4-x1.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-t4-x1.yml) | Small | AWS | [`g4dn.2xlarge`](https://aws.amazon.com/ec2/instance-types/g4/) | CentOS Stream 9 | 1 x NVIDIA Tesla T4 w/ 16 GB VRAM | `msq` | Pull Requests, Push to `main` or `release-*` branch | No |
-| [`e2e-nvidia-a10g-x1.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-a10g-x1.yml) | Medium | AWS |[`g5.4xlarge`](https://aws.amazon.com/ec2/instance-types/g5/) | CentOS Stream 9 | 1 x NVIDIA A10G w/ 24 GB VRAM | `mf` | Manually by Maintainers | No |
-| [`e2e-nvidia-l40s-x4.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-l40s-x4.yml) | Large | AWS |[`g6e.12xlarge`](https://aws.amazon.com/ec2/instance-types/g6e/) | CentOS Stream 9 | 4 x NVIDIA L40S w/ 48 GB VRAM (192 GB) | `e2e.sh` | `l` | Manually by Maintainers, Automatically against `main` branch at 11AM UTC | Yes |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| [`e2e-nvidia-t4-x1.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-t4-x1.yml) | Small | AWS | [`g4dn.2xlarge`](https://aws.amazon.com/ec2/instance-types/g4/) | CentOS Stream 9 | 1 x NVIDIA Tesla T4 w/ 16 GB VRAM | `basic-workflow-tests.sh` | `msq` | Pull Requests, Push to `main` or `release-*` branch | No |
+| [`e2e-nvidia-a10g-x1.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-a10g-x1.yml) | Medium | AWS |[`g5.4xlarge`](https://aws.amazon.com/ec2/instance-types/g5/) | CentOS Stream 9 | 1 x NVIDIA A10G w/ 24 GB VRAM | `e2e-ci.sh` | `m` | Pull Requests, Push to main or release-* branch | No |
```

Comment:
Looking at the coverage matrix below, it looks like the only difference in what's getting run is `full` training instead of `simple`. It also looks like if you're using an instance with a GPU you could serve models with vLLM. 

If those are the only two things that are the difference between the two T-shirt sizes, we could just write functional tests that run both of those things on every PR, and run the medium size e2e every day or so.

Reply:
SGTM

Reply:
This has been updated from an in-person convo today